web/src/settings_org.js

```diff
@@ -1109,8 +1174,23 @@ function check_maximum_valid_value($custom_input_elem, property_name) {
     return setting_value <= MAX_CUSTOM_TIME_LIMIT_SETTING_VALUE;
 }
 
-function enable_or_disable_save_button($subsection_elem) {
-    const time_limit_settings = [...$subsection_elem.find(".time-limit-setting")];
+function is_valid_url(string) {
```

Comment:
A better variable here would be `jitsi_url` and in case we decide to move this functions to a common file where it would be used by other parts of the code, then `url` would be better than string I guess.

Reply:
Another way to validate URL would be to use a regex but I am not sure which method will be efficient as we would check the validity of the input everytime the text in the input is changed.